### ISSUE TITLE
Mapping summary error

### ISSUE THREAD

**Issue by Fred-White94** [BleekerLab/snakemake_rnaseq#8]
Generates this error:
Error in rule generate_mapping_summary:
    jobid: 63
    output: results/mapping_summary.tsv
    shell:
        python scripts/generate_mapping_summary.py results/ results/mapping_summary.tsv
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

sometimes blocks execution of DESeq_normalisation rule

**Comment by mgalland** [comment]
Did you test this with the .fastq test samples?
```
sample	fq1	fq2
Cultivar-01	.fastq/Cultivar-01_R1.subsampled.fastq	.fastq/Cultivar-01_R2.subsampled.fastq
Cultivar-02	.fastq/Cultivar-02_R1.subsampled.fastq	.fastq/Cultivar-02_R2.subsampled.fastq
```

**Comment by Fred-White94** [comment]
Yes, with the example files it works with no error. however with my own experiment it sometimes produces this error. Perhaps again due to the filesystem latency ?..

**Comment by mgalland** [comment]
I think I fixed it in commit 7de92179fca5812689ff7dcca53f87ccd0339bca
Needs to be confirmed so I leave the issue open for now.

**Comment by mgalland** [comment]
Seems solved. Re-open if necessary
